zerver/lib/embedded_docs.py

```diff
@@ -0,0 +1,22 @@
+from django.template import loader
+import os
+
+from zerver.models import EmbeddedDocArticle
+
+def import_markdown_directory(directory: str) -> None:
+    """
+    Imports and saves a Markdown directory into a database.
+    Useful for setting up Postgres-based search of those directories.
+    """
+    articles = []
+    for entry in os.listdir("templates/" + directory):
+      if os.path.isfile("templates/" + directory + entry) and entry.endswith(".md"):
+        article_entry = {
+          "directory" : directory,
+          "title" : entry,
+          "body" : loader.get_template(directory + entry).render(),
```

Comment:
This should render the markdown too, instead of just the jinja (so that we can get the `include` text, in case people want to search that)